site/docs/blog/posts/2024-08-02-polaris.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2024-08-02
+authors:
+  - snazy
+---
+
+# Open Source Polaris announcement
+
+As many of you have already heard, the Polaris repository is now open source under the Apache License 2. The
+[announcement](https://www.snowflake.com/blog/polaris-catalog-open-source/) mentions "The team at Dremio is excited to
+help bring the various functions and capabilities of Nessie into the Polaris project.". Let me elaborate on what this
+means for now and the foreseeable future.
+
+The Nessie project was started about 4 years ago and has attracted a lot of users creating a big community around it.
+We, the Nessie OSS team, are really proud of this development. We also look forward to further strengthening the
+community around Nessie.
+
+The intent is to contribute Nessie’s <u>capabilities</u>, like Catalog Level Versioning, Git-like semantics, multi-table
```

Comment:
nit: not sure thi emphasis actually works in .md

Reply:
HTML works